packages/python-packages/apiview-copilot/tsp/main.tsp

```diff
@@ -112,7 +112,13 @@ namespace ApiViewCopilot {
   /** A review comment generated by Copilot */
   model ReviewComment {
     /** The applicable guideline IDs relevant to this comment */
-    ruleIds: string[];
+    guidelineIds: string[];
+
+    /** The applicable memory IDs relevant to this comment */
+    memoryIds: string[];
+
+    /** Whether the comment is entirely or in part formed from generic guidance */
+    is_generic: boolean;
```

Comment:
@AlitzelMendez please note the change in the schema here that need to be incorporated into APIView.